fmriprep/cli/sample_openfmri.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+A tool to sample OpenfMRI datasets using a datalad installation
+
+Please run this first: ::
+
+    # install openfmri dataset
+    datalad install -r ///openfmri
+
+    # get all sidecar files
+    datalad get -J8 openfmri/ds*/*_bold.json
+    datalad get -J8 openfmri/ds*/*T1w.json
+    datalad get -J8 openfmri/ds*/*T2w.json
+
+    # list subjects
+    cd openfmri
+
+
+"""
+
+import os
+import glob
+import numpy as np
+
+
+def get_parser():
+    """Build parser object"""
+    from argparse import ArgumentParser
+    from argparse import RawTextHelpFormatter
+
+    parser = ArgumentParser(
+        description='OpenfMRI participants sampler, for FMRIPREP\'s testing purposes',
+        formatter_class=RawTextHelpFormatter)
+
+    parser.add_argument('openfmri_dir', action='store',
+                        help='the root folder of a the openfmri dataset')
+
+    # optional arguments
+    parser.add_argument('-D', '--datalad_fetch', action='store_true', default=False,
+                        help='download sampled subjects')
+    parser.add_argument('-o', '--output-file', action='store', help='write output file')
+    parser.add_argument('-n', '--num-participants', action='store', type=int, default=4,
+                        help='number of participants randomly selected per dataset')
+    parser.add_argument('--njobs', action='store', type=int, help='parallel downloads')
+    parser.add_argument('--seed', action='store', type=int, default=20170914,
+                        help='seed for random number generation')
+
+    return parser
+
+
+def main():
+    """Entry point"""
+    thispath = os.getcwd()
+    opts = get_parser().parse_args()
+    np.random.seed(opts.seed)
+
+    out_file = None
+    if opts.output_file is not None:
+        out_file = os.path.abspath(opts.output_file)
+
+    os.chdir(opts.openfmri_dir)
+    all_sub = sorted(glob.glob('ds*/sub-*'))
+    datasets = {}
+    multises = []
+    for subj in all_sub:
+        ds = subj.split('/')[0]
+        if os.path.isdir(os.path.join(subj, 'anat')) and os.path.isdir(os.path.join(subj, 'func')):
+            if ds not in datasets:
+                datasets[ds] = []
+            datasets[ds] += [os.path.basename(subj)]
+            continue
+
+        if (glob.glob(os.path.join(subj, 'ses-*', 'anat')) and
+                glob.glob(os.path.join(subj, 'ses-*', 'func'))):
```

Comment:
Just make this `elif`, and you can drop the `continue`s, and align the `glob` expressions.